aexpect/client.py

```diff
@@ -23,7 +23,7 @@
 import shutil
 import select
 import logging
-import subprocess
+import subprocess32 as subprocess
```

Comment:
Please don't use backport as default. Newer python should use the stock subprocess. You can use something like:

```python
if sys.version_info[0:2] < (3, 2):
    import subprocess32 as subprocess
else:
    import subprocess
```

Similarly should be treated the `setup.py` install_requires (which are currently none) and the RPM specfile.

All changes have to be in a single commit as they are atomic (you need the subprocess32 and without this change you don't need subprocess32)

Reply:
I have checked that the `python-subprocess32` rpm package is in fedora repository, but it is neither in rhel repository nor in epel repository, so we may have the problem.

Reply:
You can ignore the RPM change for now, I can send it afterwards, but the missing `python-subprocess32` in epel looks more serious :-(. @clebergnu any ideas? Do you know how hard is it to add it there? And yes, we do need to add it to SPEC file even when it's in `setup.py`.

Reply:
The only (sane) way to solve this that I can think of is to attempt to get `subprocess32` on older python and in case it fails use your simplified backport. In SPEC-file I can make the `subprocess32` dependency conditional based on RHEL/Fedora.

That way on py3.2+ we'll get the up-to-date subprocess, on py3.1- we'll get subprocess32 if available and custom backport in case it's not available (RHEL from RPM)

Reply:
I see the package is ready for new rhel releases..

Reply:
btw, I see the following content in `subprocess32.python3_redirect.__init__`
```python
"""When installed as subprocess32, this sets up a redirect to subprocess."""

import subprocess
import sys
if sys.version_info[:2] < (3,3):
    raise ImportError('Ancient Python 3 versions are not supported.')
# Doing this could crash some older Python interpreters due to the module
# reference going away before the import is complete?
sys.modules['subprocess32'] = subprocess
```
So that means it does not even support py3.1- either.

Reply:
Well in Avocado we support py3.4, we can do the same in aexpect and simply use `if sys.version_info[0] < 3`.